clj-refactor.el

```diff
@@ -1511,7 +1511,7 @@ front of function literals and sets."
   (interactive)
   (insert "/")
   (when (and cljr-magic-requires
-             (looking-back cljr--magic-requires-re 6))
+             (looking-back (cljr--magic-requires-re) 6))
```

Comment:
Yeah, I was confused, since I have this function in expand-region:

```el
(defun er/looking-back-max (regexp count)
  "Version of `looking-back' that only check COUNT chars back."
  (looking-back regexp (max 1 (- (point) count))))
```

which gives the API I thought I was using. :P